src/domain/event/EventSummaryPage.tsx

```diff
@@ -183,6 +220,13 @@ const EventSummaryPage: React.FC = () => {
               </div>
 
               <div className={styles.summarySection}>
+                {!isEventDraft && isInternalEnrolment && (
+                  <div style={{ display: 'flex', justifyContent: 'flex-end' }}>
```

Comment:
2 rows upper there is a styles-object / css-file used, so why is there inline styles here?

Reply:
Decided to use style props here because it is just small helper div to get button aligned to the right. But one could argue that classNames should be use to remain consistent. Not sure which is the best solution here :)

Reply:
Changed to className